draft-ietf-tls-wkech.txt

```diff
@@ -131,157 +139,155 @@ Internet-Draft           Well-Known URI for ECH             October 2022
    o  ECHConfigList values are "current" for an hour, published with a
       TTL of 1800, and remain usable for 3 hours from the time of
       generation
-   o  Web server has a set of "backend" sites - the DNS name for each
-      such site is here represented as $BACKEND, which will end up as an
-      SNI value to be encrypted inside an ECH extension
-   o  Web server has a "front-end" site ($FRONT), where $FRONT will
-      typically be the DNS name used in the ECHConfigList public_name
-      field for ECHConfig version 0xff0d
-   o  A cronjob creates creates a JSON file for each backend site at
-      https://$FRONT/.well-known/ech/$BACKEND.json
-   o  Each JSON file contains an array with the ECHConfigList values
-      values for that particular $BACKEND as shown in Figure 1 - the
-      values in Figure 1 with ellipses are the values we want to
-      eventually see in the DNS
+   o  A cronjob creates creates a JSON file containing the ECHConfigList
+      site at https://$ORIGIN/.well-known/dns-zone-info
+   o  The JSON file contains an array with the ECHConfigList values
+      values as shown in Figure 1 - the values in Figure 1 with ellipses
```

Comment:
nit: IIRC, the values have ".." which is one dot short of an ellipsis.

Reply:
I've rewritten this list to reflect the cache-lifetime-driven behavior in this PR.